data/data/azure/bootstrap/variables.tf

```diff
@@ -68,3 +68,8 @@ variable "nsg_name" {
   type        = string
   description = "The network security group for the subnet."
 }
+
+variable "preexisting_network" {
```

Comment:
why not use the `nsg_name` which will be empty (see here https://github.com/openshift/installer/pull/2441/files#diff-5a55d1d520f80cdfdd21c6cb9c65d9eeR26) to decide if a rule needs to be added..?

Reply:
Agree and updated!